server/src/main/java/com/nukkitx/server/NukkitServer.java

```diff
@@ -151,9 +151,15 @@
     static {
         NAME = NukkitServer.class.getPackage().getImplementationTitle();
         Package mainPackage = NukkitServer.class.getPackage();
-        API_VERSION = SemVer.fromString(mainPackage.getSpecificationVersion().replace("-SNAPSHOT", ""));
-        MINECRAFT_VERSION = SemVer.fromString(mainPackage.getImplementationVersion().replace("-SNAPSHOT", ""));
-        NUKKIT_VERSION = mainPackage.getImplementationVendor();
+        if (mainPackage == null || mainPackage.getSpecificationVersion() == null)    {
+            API_VERSION = SemVer.fromString("1.0.0");
+            MINECRAFT_VERSION = SemVer.fromString("1.2.3");
+            NUKKIT_VERSION = "4.5.6";
```

Comment:
This should be find for now but an alternative solution should be found considering the API version is used to validate if a plugin is compatible.